drivers/dma/dma_stm32.c

```diff
@@ -286,6 +313,9 @@ static int dma_stm32_configure(struct device *dev, u32_t id,
 	stream->callback_arg    = config->callback_arg;
 	stream->src_size	= config->source_data_size;
 	stream->dst_size	= config->dest_data_size;
+#ifdef CONFIG_DMAMUX_STM32
+	/* the stream->mux_channel is fixed by the dma init */
+#endif /* CONFIG_DMAMUX_STM32 */
```

Comment:
I'm doubtful about comments under compile flags. And potentially some static code analyzer may complain.
Can you just reword this part to make clear that incase of DMAMUX stream->mux_channel is fixed by the dma init ?

Reply:
moving this comment directly in the dma_init function